api/config/cdr_config_preprod.json

```diff
@@ -67,6 +67,20 @@
       "hasFitbitData": true,
       "hasCopeSurveyData": true,
       "accessTier": "registered"
+    },
+    {
+      "cdrVersionId": 5,
+      "name": "All of Us Dataset v5 Controlled Tier",
```

Comment:
Could you rename to follow https://precisionmedicineinitiative.atlassian.net/browse/RW-6647 ?

Reply:
Done.